locate_cuda_root.sh

```diff
@@ -48,9 +48,14 @@ if [ ! -z "$1" ]; then
 elif [ ! -z "$CUDA_ROOT" ]; then
     ## Use already set value
     break
-else
+elif [ ! -z "$(which nvcc)" ]; then
     ## Heuristic: $CUDA_ROOT/bin/nvcc
-    CUDA_ROOT="$(dirname $(dirname $(which nvcc)))"
+    nvcc = $(readlink -f $(which nvcc))
```

Comment:
I don't think this can work - variable assignment in posix sh requires not having a space between the variable being assigned and the equals sign.

Reply:
Oh, right! :facepalm: